webpack/components/TypeAhead/pf4Search/TypeAheadSearch.js

```diff
@@ -7,7 +7,6 @@ import keyPressHandler from '../helpers/helpers';
 import TypeAheadInput from './TypeAheadInput';
 import TypeAheadItems from './TypeAheadItems';
 import commonSearchPropTypes from '../helpers/commonPropTypes';
-import './TypeAheadSearch.scss';
```

Comment:
Fixes the wonky X button for search, it must of been fixed in a patternfly update